README.md

```diff
@@ -6,88 +6,21 @@ To add a conference to the list, please [contribute](#contribute)!
 
 If you would like to be notified of future conferences that would be added to the list, you could watch this repo. New additions are made via PRs and have full description of the conference so that you can view it directly in your email. ([Example PR](https://github.com/nikhita/tech-conferences-india/pull/25))
 
-## 2018
+## 2017
 
-| Conference | Date | Venue | Description |
-|------------|------|-------|-------------|
-| [Great Indian Developer Summit](http://www.developermarch.com/developersummit/index.html) | 24-27 Apr | Bengaluru | 	With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem.
-| [GopherCon India](http://www.gophercon.in/) | 09-10 Mar | Sheraton Grand, Sangamvadi, Pune | Go conference in India. Go is an open source project developed by a team at Google. |
-| [PyCon Pune](https://pune.pycon.org/2018/) | 08-11 Feb | Pune | The community will meet for two days of talks and work on upstream projects in two days of dev sprint. |
-| [Hosting and Domain Conference India](http://www.hdcon.org/) | 3 Feb | Hyderabad | Hosting and Domain Conference is the World's Top initiative to discuss on hosting, domain, blogging and web technologies. |
+[List of Conferences](year-pages/2017-conferences-list.md)
```

Comment:
I'd like to keep the current year as-is in the main README and not move it to a separate directory.

Can you move the 2017 conferences back to the README? Thanks!

Reply:
So that people can view the current year's list without any further clicks. Most of the time, they just want to view the current year.

Reply:
Yeah, that makes sense abt 2017 being in the main page itself.

am not clear about squashing commit into a single commit.

Reply:
@techsethi That's all right then, I'll use the Github UI to squash the commits.

Please make the changes for the 2017 one, I'll take care of the squashing. :smile: